src/commands/api/getConnectionString.ts

```diff
@@ -12,9 +13,9 @@ export async function getConnectionString(treeItemId: string): Promise<string> {
         throw new Error(`Couldn't find the database node in Cosmos DB with provided Id: ${treeItemId}`);
     }
 
-    if (node instanceof MongoDatabaseTreeItem) {
+    if (node instanceof MongoDatabaseTreeItem || node instanceof DocDBDatabaseTreeItem) {
```

Comment:
I would do `instanceof DocDBDatabaseTreeItemBase` since that's the class you implemented this on